aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -139,6 +354,25 @@ static inline Tensor _fft(const Tensor &self, const int64_t signal_ndim,
   return output;
 }
 
+// Wrapper to preserve the historic signature of _fft_with_size
+// This is only used for torch-script back-compat and the new signature with
```

Comment:
nit: "NOTE: This is only used for torchscript backwards compatibility, and the new signature..."